/**:
  ros__parameters:
    voxel_size_x: 1.0
    voxel_size_y: 1.0
    voxel_size_z: 1.0
